files/image_config/kubernetes/scripts/pod_readiness.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+
+# logger -p local0.notice -t image "daemon:$1 image:$2"
```

Comment:
I do not understand this script, seems do nothing here?

Reply:
Kuberenetes supports start, stop, [liveness & readiness](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) probes. I have provided pod_hook.sh with start/stop args that a manifest could use as start/stop. 
This script is more a place holder for readiness. Unlike start/stop, which gets called once during container lifetime, the readiness is called periodically, with default being every 10 seconds. Hence I intentionally left it empty.

Reply:
What if we remove the placeholder? We can add it later if needed.

---
In reply to: [403756889](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r403756889) [](ancestors = 403756889)

Reply:
ok. Let me remove it.